php-fpm/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM php:7.2-fpm
+
+RUN apt-get update
+
+RUN apt-get install -y zlib1g-dev libpq-dev git libicu-dev libxml2-dev \
```

Comment:
I just tried it, you can definitely remove `zlib1g-dev` and `libxml2-dev` as long as you remove `zip` and `xml` from the `docker-php-ext-install`